### ISSUE TITLE
imports not used when using multi-root workspaces for website development in VSCode

### ISSUE THREAD

**Issue by taras** [thefrontside/effection#893]
[VSCode Deno plugin does not read `deno.json`](https://github.com/denoland/vscode_deno/issues/938) when used in multi-root workspace configuration. `www/deno.json` has imports path for Effection in other dependencies which Deno LSP does not know how to resolve which causes the following error

<img width="878" alt="image" src="https://github.com/thefrontside/effection/assets/74687/6560fa81-7b3e-48cc-bb04-c7763aa80e38">

**Comment by cowboyd** [comment]
@taras Does [the workaround](https://github.com/denoland/vscode_deno/issues/938#issuecomment-1734106460) in that issue work?

**Comment by taras** [comment]
No, because it's not the same issue. Our issue isn't that `deno.json` is in a folder. The problem is that it only reads the `deno.json` of the root workspace. The workaround in #894 is to make `www` root, which automatically reads the `deno.json` with imports relying on the fact that the core doesn't have any imports.

**Comment by jbolda** [comment]
The docs site has been moved into a separate repo (and also I think deno now supports this, but either way) so this can be closed,
